config/environment.js

```diff
@@ -7,8 +7,8 @@ module.exports = function(environment) {
     baseURL: '/',
     locationType: 'auto',
 
-    authBaseUri: "http://localhost:4000",
-    apiBaseUri: "http://localhost:4001",
+    authBaseUri: "http://auth.aptible.foundry.io",
+    apiBaseUri: "http://api.aptible.foundry.io",
```

Comment:
Good catch, I updated to use ENV vars where available, so I don't keep trying to sneak this in...